docs/guides/desktop/mate_installation.md

```diff
@@ -12,55 +12,145 @@ tags:
 
-## Final Steps
+    At this point, you can use the MATE environment and see if you like it. Once you've decided that you would like to use it permanently, double-click the option to **Install to Hard Drive.**
 
-Now that we have everything we need installed, the next thing we need to do is set the minimal install to boot into the Graphical User Interface (GUI). We can do this by entering:
+    This will start a pretty familiar installation process for those who have installed Rocky Linux before. In the following screen, we've highlighted things that you *may* wish to change.
 
-`sudo systemctl set-default graphical.target`
+    ![mate_install](images/mate_install.png)
 
-Now just keep your fingers crossed and reboot:
+    1. **Keyboard** Take a look at this option and make sure that it matches up to the keyboard layout that you use.
+    2. **Time & Date**  Make sure this matches up to your time zone.
+    3. **Installation Destination** You'll need to click into this option, even if it is just to accept what is already there.
+    4. **Network & Host Name** Verify that you have what you want here. As long as the network is enabled, you can always change this later if you need to.
+    5. **Root Password** Go ahead and set a root password. Remember to save this somewhere safe (password manager), particularly if it isn't something you will be using often.
+    6. **User Creation** Definitely create at least one user. If you want the user to have administrative rights, remember to set this option when creating the user. 
+    7. **Begin Installation** Once all of the settings have been set or verified, go ahead and click this option.
 
-`sudo reboot`
+    Once you do step 7, the installation process should start installing packages, as in the below screenshot:
 
-Next, click on your user name on the screen, but before you enter your password and login, click on the gear icon to the left of the "Sign in" option. Select "MATE" from the available desktop choices and then you can enter your password and login. Future logins will remember your selection.
+    ![mate_install_2](images/mate_install_2.png)
 
-## Conclusion
+    After the installatin to the hard drive is complete, you'll be faced with the following screen:
 
-Some people are not satisfied with the newer GNOME implementations or simply prefer the older MATE GNOME 2 look and feel. For those people, getting MATE installed in Rocky Linux will provide a nice, stable alternative.
+    ![mate_install_final](images/mate_install_final.png)
+
+    Go ahead and click **Finish Installation**
+
+    At this point the OS will reboot, and you should remove your boot media. When the OS comes up for the first time, there is a licensing agreement screen:
+
+    ![eula](images/eula.png)
+
+    Go ahead and click the **I accept license agreement** check box, and then click **Done.** This will get you to the final screen as far as the installation is concerned. On this screen, click **Finish Configuration**.
```

Comment:
```suggestion
    Go ahead and click the **I accept the license agreement** check box, and then click **Done.** This will get you to the final screen as far as the installation is concerned. On this screen, click **Finish Configuration**.
```